data/settings.js

```diff
@@ -1154,18 +1306,18 @@ The configured value must lie between [[setting,sieve_vacation_min_period]]
 and [[setting,sieve_vacation_max_period]].`
 	},
 
-	sieve_vacation_dont_check_recipient: {
+	sieve_vacation_check_recipient: {
 		tags: [ 'sieve', 'sieve-vacation' ],
 		plugin: 'sieve',
-		default: 'no',
+		default: 'yes',
 		values: setting_types.BOOLEAN,
 		seealso: [ '[[link,sieve_vacation]]' ],
 		text: `
-This disables the checks for implicit delivery entirely. This means that
-the vacation command does not verify that the message is explicitly
-addressed at the recipient.
+This setting determines whether the checks for implicit delivery are performed.
+If this is skipped, this means that the vacation command does not verify that
+the message is explicitly addressed at the recipient.
 
-Use this option with caution. Specifying \`yes\` will violate the Sieve
+Use this option with caution. Specifying \`no\` will violate the Sieve
 standards and can cause vacation replies to be sent for messages not
 directly addressed at the recipient.`
 	},
```

Comment:
sieve_vacation_max_period talks about 0 value being no upper limit, and it being the default. This should be changed to "infinite" instead. Not sure if 0 should be an error, or if it should really be the number 0. In any case, 0 shouldn't be same as infinite.

Looking at other such issues, I found this weirdness in the code:

```
.max_cpu_time = 0, /* FIXME: svinst->env_location == SIEVE_ENV_LOCATION_MS */
```